.travis.yml

```diff
@@ -13,9 +13,15 @@ install:
   # At this moment this plugin has overlapping dependencies than qiita so installing everything in the same environment
   - conda create --yes -n qiita_env python=2.7 pip nose flake8
     pyzmq networkx pyparsing natsort mock future libgfortran
-    'pandas>=0.18' 'scipy>0.13.0' 'numpy>=1.7' 'h5py>=2.3.1'
+    'pandas>=0.18' 'scipy>0.13.0' 'numpy>=1.7' 'h5py>=2.3.1' matplotlib
   - source activate qiita_env
+  - export CONDA_BIN=/home/travis/miniconda3/bin
```

Comment:
Not sure if you are using this anymore, but this no longer running on travis so this path is incorrect. Check [this line](https://github.com/qiita-spots/qp-shotgun/pull/4/files#diff-354f30a63fb0907d4ad57269548329e3R33) for an idea on how to fix it, unless you don't need anymore, in which case can be removed.